submissions/7SergSmith7/Friends-app/app.js

```diff
@@ -0,0 +1,142 @@
+import "../css/vendor/normalize.css";
+import "../css/styles.css";
+
+const USERS_URL =
```

Comment:
You can keep this URL in `getContacts`, because this is the only URL that is used by this function.